### ISSUE TITLE
Incomplete response body when deploying with nginx

### ISSUE THREAD

**Issue by li-thy-um** [midori-rb/midori.rb#151]
error message
```
Failure/Error: puts JSON.pretty_generate(Oj.load(r.body))

     Oj::ParseError:
       quoted string not terminated at line 1, column 86807 [parse.c:337]
```

headers
```
{:server=>"nginx", :date=>"Wed, 31 Jan 2018 07:32:38 GMT", :content_type=>"application/json", :content_length=>"87904", :connection=>"close"}
```

nginx.conf
```
server {
        listen 6199;
        server_name default;
 
        proxy_next_upstream error timeout http_502 http_504;

        location / {
                proxy_pass http://localhost:3000/;
        }
}
```

**Comment by dsh0416** [comment]
Does the body contains non-ASCII characters like Chinese Characters? It seems that there be a length calculation error when parsing.

**Comment by dsh0416** [comment]
Wait, why the header contains `content_length` instead of `content-length`?

**Comment by dsh0416** [comment]
Bug confirmed. It seems that there is a bug in the test file ﻿concealed the bug. #152

**Comment by dsh0416** [comment]
Fixed in #152. This is a pretty serious bug. Thank you for your contribution.

**Comment by li-thy-um** [comment]
I use gem `rest-client` to do request and check response, so the headers are like that.

**Comment by li-thy-um** [comment]
confirm fixed on v0.5.2
